ui/lib/core/addon/components/info-table-row.js

```diff
@@ -33,9 +34,11 @@ export default Component.extend({
   isVisible: or('alwaysRender', 'value'),
 
   alwaysRender: false,
+  renderBlank: false,
```

Comment:
Should this also be documented? I am a non-knower of the conventions here though 😆

Reply:
Making a ticket for this.